### ISSUE TITLE
Sample example link in ReadMe file gives 404

### ISSUE THREAD

**Issue by punamdahiya** [randdusing/cordova-plugin-bluetoothle#330]
Sample example link https://github.com/Microsoft/cordova-samples/tree/master/cordova-plugin-bluetoothle in cordova-plugin-bluetoothle goes to 404. Please update

**Comment by randdusing** [comment]
I don't think this exists anymore. I plan on creating my own simple example in vanilla Javascript, but until then it's best to checkout out the Angular wrapper example.
